app/forms/claims/claim/mentors_form.rb

```diff
@@ -53,7 +53,7 @@ def mentor_trainings
         mentor_id:,
         provider_id: claim.provider_id,
         claim_id: claim.id,
-      )
+      ).tap(&:set_training_type)
```

Comment:
This will attempt to re-set the training type each time it is called. I _think_ Rails has some _magic_ to avoid `update` calls to values that are unchanged, but I don't think it is that clever in the case of assigning values and then calling `#save`.

Maybe renaming to `#set_initial_training_type` and using `||=` addresses this.

Nevertheless, I don't see it as a blocker.

Nicely done everyone!

Reply:
Active Record will avoid re-saving the model if an attribute has been assigned but its value hasn't changed. So we should be all good! 🙂